.evergreen/compile_cs.sh

```diff
@@ -26,8 +22,12 @@ if [ "$OS" == "Windows_NT" ]; then
 
     # Make dotnet happy over ssh
     export DOTNET_CLI_HOME=$(cygpath -w "${evergreen_root}/dotnet_home")
+else
+    export PATH=$PATH:/usr/share/dotnet
 fi
 
+dotnet_tool=$(which dotnet)
```

Comment:
this line uses the PATH which we set above, so the calling order is slightly updated